### DISCUSSION TITLE
No scrollbar when the Select has too many options

### DISCUSSION THREAD

**Discussion by dudduss** [csandman/chakra-react-select#240] [Q&A]
I currently have a select with 6 options, but I can't scroll through all of them unless I search through it. Is there a way to enable scrolling that I'm missing from the documentation?

```
          <ChakraReactSelect
            size="md"
            onChange={(e) => {
              setSelectedDocumentTypes(e.map((option) => option.value));
            }}
            tagVariant="solid"
            isMulti
            placeholder="Select document type(s)"
            options={documentTypes.map((type) => ({
              label: type,
              value: type,
            }))}
``` 

<img width="656" alt="image" src="https://user-images.githubusercontent.com/9653576/219849343-97d46ef2-e70a-4fc3-a83a-9981ab91df7a.png">

**Comment by csandman** [comment]
This is not a standard behavior for this package. If you look at any of the examples, scrolling is functional by default. Based on the odd border around your select, it seems like you might have some third party styles interfering with the built-in styles. The only way I could help with this is if you can recreate this issue in a CodeSandbox so I can take a look at it.
